docs/Release_notes.md

```diff
@@ -0,0 +1,126 @@
+JPO-ODE Release Notes
+----------------------------
+
+Version 1.3.0, released Mar 30th 2023
+----------------------------------------
+
+### **Summary**
+The updates for jpo-ode 1.3.0 release includes enhancements such as support for Confluent Cloud, better debugging with the C++ submodules, new JSON output schemas, and improvements for all J2735 message type ser/des. Along with the enhancements below, several bug fixes and CI related enhancements are included in this release.
+
+Enhancements in this release:
+-	Issue 466: Removes duplicated J2735 payload data due to Jackson serialization default behavior. Adds annotations to payload classes. Resolves BSM coreData support issues.
+-	Issue 468: Resolves TIM ingestion issues created by the previous removal of the gson libraries.
+-	Issue 470: Adds unit conversions for MAP and SRM messages for Position3D, heading and speed. This ensures they are compliant with the rest of the jpo-ode conversions.
+-	Issue 471: Updates logging calls to reduce logging at the info level.
+-	Issue 472: Resolves bug with J2735TransmissionAndSpeedBuilder values being populated incorrectly. Resolves bug for BSM crumbData for unknown values.
+-	Issue 474: Provides JSON output schemas for the jpo-ode message types.
+-	PR 477: Adds Confluent Cloud support by supporting SASL authenticated Kafka brokers.
+-	PR 478: Bumps kafka-clients from 0.10.1.0 to 0.10.2.2
+	PR 482: Adds VSCode support for debugging C++ submodules within the jpo-ode project.
+-	Issue 485: Updates the jpo-ode SPaT output to better support the J2735 established enum values.
+-	Issue 491: Updates schemas to reflect current message output structure of the jpo-ode message types. Removes null values from jpo-ode output messages.
+-	Issue 493: Fixes SRM bug for properly deserializing multiple requests in a single SRM message and handling zero requests.
+-	PR 494: Bumps logback-core from 1.2.8 to 1.2.9
+-	PR 497: Updates base image to eclipse-temurin:11-jre-alpine rather than the deprecated openjdk:8-jre-alpine.
+-	PR 498: Adds the GitHub actions workflows for both docker hub build and to run sonar scan analysis for publishing static reports on sonar cloud dashboard.
+-	PR 500: Modifies TimDepositController class to allow unknown properties when converting a JSON object to a POJO, resolving an issue where TIMs with additional RSU properties were causing process failures.
+
+Known Issues:
+-	One of the tests for the TimDepositController class is failing due to invalid test input. Specifically, the test expects malformed json but is currently being given valid json.
+-	The lane type in map messages isn’t handled correctly at this time.
```

Comment:
This has now been proven to be incorrect. LaneType is confirmed to be handled properly according to the J2735 standard. An incorrectly configured MAP message is now believed to be the culprit.